pkgs/build-support/build-fhsenv-bubblewrap/default.nix

```diff
@@ -239,7 +242,7 @@ in runCommandLocal name {
   };
 } ''
   mkdir -p $out/bin
-  ln -s ${bin} $out/bin/${pname}
+  ln -s ${bin} $out/bin/${name}
```

Comment:
> Since `buildFHSEnvBubblewrap` would also link its output to `ln -s ${bin} $out/bin/${pname}` and some packages relied on this behavior, we change this line to `ln -s ${bin} $out/bin/${name}` so that existing packages are not broken.

But what if an existing package uses `pname` (without version) and relies on it? Is there any package that relies on this?

Reply:
The link destination has always been `$out/bin/${CORRECT_PNAME-CORRECT_VERSION}`. Before https://github.com/NixOS/nixpkgs/commit/055b69b76dc66c24d7b948b2c913eaf438194fbd, `buildFHSEnvBubblewrap` should have returned derivations without a `pname`, only `name`, so packages should not be able to rely on that attribute. Besides, I it's easier for packages to just use what they pass to `buildFHSEnv` as `pname`.

But maybe a better way to test is to just test to rebuild all affected packages.